openedx/features/pakx/lms/overrides/utils.py

```diff
@@ -370,7 +373,7 @@ def get_progress_information(request, course_key):
                      output_field=IntegerField())), 0
         ),
         problem=Coalesce(
-            Sum(Case(When(block_type__in=['problem', 'edly_carousel'], then=1), default=0,
+            Sum(Case(When(block_type__in=['problem', PROBLEM_BLOCK_TYPES], then=1), default=0,
```

Comment:
why didn't you add `problem` in `PROBLEM_BLOCK_TYPES`?

Reply:
it's a mistake, correcting